operatorhub/openshift/release-artifacts/bundle/manifests/openshift-pipelines-operator-rh.clusterserviceversion.yaml

```diff
@@ -908,6 +908,10 @@ spec:
                   value: registry.redhat.io/openshift-pipelines/pipelines-serve-tkn-cli-rhel8@
                 - name: IMAGE_CHAINS_TEKTON_CHAINS_CONTROLLER
                   value: registry.redhat.io/openshift-pipelines/pipelines-chains-controller-rhel8@
+                - name: IMAGE_RESULTS_TEKTON_API
```

Comment:
If we follow the other ones, it would look like `IMAGE_RESULTS_TEKTON_RESULTS_API` and `IMAGE_RESULTS_TEKTON_RESULTS_WATCHER` ?
Not sure why the duplication is there, but we might want to keep the same type of naming to be consistent ?

Reply:
Done